incubator/kubernetes-tagger/templates/clusterrole.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "kubernetes-tagger.fullname" . }}
+  labels:
+    app: {{ template "kubernetes-tagger.name" . }}
```

Comment:
please use the new labels you can find the new labels here: https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md
also when you use the latest helm to start the project it already creates using the new labels

Reply:
It is fixed now. I've started the chart with helm 2.11.0, that's why :).